php/libraries/Database.class.inc

```diff
@@ -895,7 +895,7 @@ class Database
             $query = "SELECT * FROM $table WHERE $where";
 
             // select the database data
-            $this->select($query, $oldRows);
+            $oldRows = $this->pselect($query, array());
```

Comment:
I think this should be in a separated commit. Is it required for your PullRequest ?